### DISCUSSION TITLE
how do i show errors for mistake made to xdp.c?

### DISCUSSION THREAD

**Discussion by kolinfluence** [cilium/ebpf#881] [Q&A] [answered]
how do i show errors for mistake made to xdp.c?
basically when i purposely make a coding error in xdp.c, how can i make it show errors?

with reference to https://github.com/cilium/ebpf/discussions/877

how do i build the xdp.c to show error because running
go run github.com/cilium/ebpf/cmd/bpf2go -cc $BPF_CLANG -cflags $BPF_CFLAGS bpf xdp.c -- -I../headers

shows
```
Error: missing package, are you running via go generate?
exit status 1
```

**Comment by lmb** [comment, marked as answer]
@ultperf please stop creating discussions covering the same topic over and over. As both Timo and the error message suggest, you aren't invoking go generate. go build != go generate. See https://go.dev/blog/generate.
